src/kv/MemDB.h

```diff
@@ -132,12 +133,16 @@ class MemDB : public KeyValueDB
       std::pair<string, bufferlist> m_key_value;
       btree::btree_map<std::string, bufferptr> *m_btree_p;
       std::mutex *m_btree_lock_p;
+      uint64_t *btree_seq;
+      uint64_t current_seq;
```

Comment:
Rename it to something : global_seq_no and my_seq_no or this_seq_no. Current seq_node is confusing .